Check mutation's status in mutation logs

bmonjoie opened the pull request:
Following the previous PR, it was brought to my attention that the mutations were asynchronous, and the mutation's status was to be checked in the mutation logs.

This PR adds a new use case to check for the different mutations and propagate their actual status.

We also noticed that the enrolment's creation/modification was sub-optimal and could lead to unstable states within the mobile application. To fix that, we created a new server-side mutation called `mobileEnrollment`, and added it to the mobile application.
This reduces the risk of an unstable state within the mobile application and significantly lowers the complexity on the mobile side.

Finally, this PR addresses a regression with the insuree's pictures.

bmonjoie committed:
Fetch mutation status from mutation logs

- Update schema.graphqls.
- Add new CheckMutation query.
- Add `clientMutationUuid` and `clientMutationLabel` to mutation queries.
- Fix some queries.
- Remove call to `UpdateFamily` as this is not needed and doesn't work.
- Call to `CheckMutation` in mutation queries.
- Fix syntax error in `exact.js` which broke the whole application.

OPENIMIS-75

bmonjoie committed:
Fix create policy

OPENIMIS-75

bmonjoie committed:
Use the new `mobileEnrolment` mutation

OPENIMIS-75

bmonjoie committed:
Fix pictures

OPENIMIS-75

tdethier commented:
@delcroip @dragos-dobre As Benjamin explained, the purpose of this PR is to fix the issues we've had with the policy app without the REST API. 

The `mobileEnrollment` endpoint he mentioned is part of a new module called `openimis-be-mobile`, which is [available here](https://github.com/BLSQ/openimis-be-mobile_py/).
Basically, we renamed the `controls` module and we'll add there all the necessary things for using the mobile apps. In order to avoid any issues with current implementations, we recommend:

- Forking the `mobile` module into the openIMIS organization
- Deprecating the `controls` module and updating its description to explain that it is deprecated
- Installing the `mobile` module on the desired instances
- Removing the `controls` module where it was already installed

Our next step will now be to make sure that the renewals and feed-backs are working properly.

bmonjoie committed:
Fix renewals

OPENIMIS-75

bmonjoie committed:
Fix moving renewal files

OPENIMIS-75

tdethier committed:
Fixed Location picker

Fixed a bug with the location picker while being an enrollment officer.
Added FR translation values for Niger variant.

bmonjoie committed:
Fix QRCode scanning

OPENIMIS-75
